final_prep/mod_zero_hero.rb

```diff
@@ -74,11 +126,29 @@
 
 # - Create 2 instances of your SuperHero class
 
+priya = SuperHero.new("Priya", "attention to detail", 25)
+mason = SuperHero.new("Mason", "positivity", 29)
+
+puts priya.name
+puts priya.super_power
+puts priya.age
+puts priya.arch_nemesis
+puts priya.power_level
+puts priya.energy_level
+priya.maximize_energy
+priya.gain_power(150)
+puts priya.power_level
+puts priya.energy_level
 
 # Reflection
 # What parts were most difficult about this exerise?
+# I struggled a little with the SuperHero class and whether to use @power_level/@energy_level,
```

Comment:
Priya and I are superheros! Hooray!
This is a little tricky- those can be used interchangeably, but it is best to stick to one type of syntax per project. They both direct towards which property to change- when I call method say_name, what name is being said? My name. Not some other instance's name.